web/src/cypress/support/menu.ts

```diff
@@ -24,7 +24,11 @@ function menu(
     const format: 'mobile' | 'wide' = el.data('cy-format')
     expect(format, 'header format').to.be.oneOf(['mobile', 'wide'])
 
-    cy.wrap(sub).click()
+    // Can't just use `.click()` due to
+    // an issue with Cypress 3.6.0
```

Comment:
Does this affect `.click()` app wide? This is pretty concerning if this feature is broken on the current version 🙀

Reply:
Couldn't nail down exactly what caused it. But was still seeing failures in 3.6.1 (changelog indicated it contained a fix).

Unfortunately 3.6.1 broke `cy.request` anyhow, so reverting back to 3.4.1 for now.